src/js/components/event/confirmed-event.js

```diff
@@ -3,97 +3,70 @@
 /* eslint-disable no-unused-vars */
 import React from 'react';
 import { View, Image, Text } from 'react-native';
-import RSVPsArea from './confirmed-event/RSVPs-area';
+import rsvpsArea from './confirmed-event/rsvps-area';
```

Comment:
@minaorangina here I think you missed to change the file name. The path is does not match this file name.